src/widget/sidebar_filler.h

```diff
@@ -0,0 +1,19 @@
+#ifndef WIDGET_SIDEBAR_FILLER_H
+#define WIDGET_SIDEBAR_FILLER_H
+
+#include "graphics/arrow_button.h"
+
+arrow_button *sidebar_filler_get_arrow_buttons_speed(void);
```

Comment:
These shouldn't be exposed: code that uses the filler should not know that arrow buttons are used internally.